examples/mcp-project-reporting/README.md

```
@@ -0,0 +1,88 @@
+
+## Known Limitations
+
+* Depending on the model being used, tool calling can be unreliable.
+* Queries may have to be specific. For example, `what's new with my projects` might result in being asked to name a specific project to request status for.
+
+
+## Usage
+
+Install Llama Stack:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)